src/elastio-snap.c

```diff
@@ -1003,6 +1004,11 @@ static void elastio_snap_proc_stop(struct seq_file *m, void *v);
 static int elastio_snap_proc_open(struct inode *inode, struct file *filp);
 static int elastio_snap_proc_release(struct inode *inode, struct file *file);
 
+// wait msec value to be at least 100 msec as wait loop uses it by msleep of (100) timeout pieces
+#define ELASTIO_SNAP_WAIT_FOR_RELEASE_MSEC              500
+#define ELASTIO_SNAP_WAIT_FOR_RELEASE_TIMEOUT_PEICE_CNT 100
```

Comment:
I think this is an error, you meant to write `PIECE`, but even spelled correctly it's not clear what this means.  I think something better would be:

```suggestion
#define ELASTIO_SNAP_WAIT_FOR_RELEASE_MAX_SLEEP_COUNT 100
```